crowbar_framework/lib/crowbar/conduit_resolver.rb

```diff
@@ -106,7 +106,7 @@ def if_speed_remap
 
         sorted_ifs.each do |intf|
           speeds = cr_ohai_network[intf]["speeds"]
-          speeds = ["1g"] unless speeds # legacy object support
+          speeds = ["1g"] if speeds.nil? || speeds.empty? # legacy object support
```

Comment:
We habe rails anyway installed so why not using it? 

Just require and it will work: `require "active_support/core_ext/object/blank"`